h/models/annotation.py

```diff
@@ -149,6 +149,14 @@ def target_uri(self, value):
     def target_uri_normalized(self):
         return self._target_uri_normalized
 
+    @property
+    def target(self):
+        target = {"source": self.target_uri}
+        if self.target_selectors:
+            target["selector"] = self.target_selectors
+
+        return [target]
```

Comment:
Not sure when this list will be len > 1. But that's a separate discussion.

Reply:
I don't know. I'm guessing we're trying to meet or emulate a third party spec?